charts/prometheus/values.yaml

```diff
@@ -1297,7 +1297,7 @@ serverFiles:
       - job_name: prometheus
         static_configs:
           - targets:
-            - localhost:9090
+            - localhost:9090{{.Values.server.baseURL}}/metrics
```

Comment:
base URL encompasses scheme + host + base path if I'm not mistaken, whereas the target should be a host, we should strip away the scheme if any

Reply:
Yes, you are correct. I got confused with prefixURL. Updated.